ledger/complete/wal/checkpointer.go

```diff
@@ -314,10 +339,14 @@ func LoadCheckpoint(filepath string) (*flattener.FlattenedForest, error) {
 	if magicBytes != MagicBytes {
 		return nil, fmt.Errorf("unknown file format. Magic constant %x does not match expected %x", magicBytes, MagicBytes)
 	}
-	if version != VersionV1 && version != VersionV2 {
+	if version != VersionV1 && version != VersionV3 {
```

Comment:
we might still need to be able to read older formats to be able to migrate, I was under this impression that every time we bump the version we keep the old version readers for backward compatibility

Reply:
Yes, but the thing is - when we switched to new ledger, we "reset" the version numbering. In this PR you can see we used to write V1 as a version, and V2 was never used really. But for the sake of compatibility I bumped to V3 to avoid confustion.
https://github.com/onflow/flow-go/pull/186/files#diff-4d387a87aa55ecd3590e6cfeaa400f14d6aa9823fcbf4cded30e23c0117b767fL242
So far, the only difference between V1 and V3 is writing CRC32 checksum, which is not read/checked for V1